api/app.go

```diff
@@ -0,0 +1,32 @@
+package api
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+// App struct
+type App struct {
+	Router *mux.Router
+	host   string
+	port   int
+}
+
+func (a *App) getRouter() *mux.Router {
+	router := mux.NewRouter()
+	router.Handle("/servers/{type}", NewServersHandler(a)).Methods("GET")
```

Comment:
we should also be able to list all servers, I'd rather have an optional query string parameter:

`/servers?type=<xyz>`

Reply:
Just to make sure I got this right, so ``/servers`` would list all servers and ``/servers?type=connector`` (for example) would list all ``connector`` servers?